### ISSUE TITLE
Implement command for creating new rhetos app

### ISSUE THREAD

**Issue by VladimirZD** [Rhetos/RhetosCli#2]
Create a new template project with Rhetos binaries and a sample Nuget package with one or two .rhe scripts.

**Comment by nguyenlamlll** [comment]
Hi, are you actively working on this issue? If it's possible, I'd love to help implementing this one 😄

**Comment by VladimirZD** [comment]
Yes i am, creating new Rhetos app is working(with some minor bugss and limitations) i need few days to fix minor bugs so any1 can try it
Feel free to check out source and let me know where you would like to help..
